BoxContentSDK/BoxContentSDK/Operations/BOXAPIThumbnailDownloadDataOperation.h

```diff
@@ -0,0 +1,28 @@
+//
+//  BOXAPIThumbnailDownloadOperation.h
```

Comment:
BOXAPIThumbnailOperation

Also, should we limit it to thumbnails? It's really about small data operations that you might need quickly and in bulk - you could imagine that also being metadata (which we may one day need while scrolling through items for  cell details), or audio mini snippets, or something else.

What do you think of BOXAPISmallDataOperation or something like that?